hathor/transaction/transaction.py

```diff
@@ -547,7 +558,7 @@ def verify_spent_reward(self, block: Block) -> None:
             assert len(tips) > 0
             tip = self.storage.get_transaction(tips[0])
             assert tip is not None
-            assert self.timestamp > tip.timestamp
+            assert self.timestamp > tip.timestamp  # XXX: should this be an assert? fails on a test
```

Comment:
I guess so since `tip` comes from `get_best_block_tips(self.timestamp - 1)`.

Reply:
Fixed.

Reply:
Fixed in #292 .